### ISSUE TITLE
Directory doesn't exist.

### ISSUE THREAD

**Issue by skinnyjames** [jarofghosts/jung#6]
When running the command 
```
jung -r /var/log -- echo \$JUNG_FILE changed!"
```
I get this error
```
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: !! Root dir `/var/log` does not exist !!
    at EventEmitter.Jung$start [as start] (/usr/lib/node_modules/jung/index.js:117:9)
    at Object.<anonymous> (/usr/lib/node_modules/jung/bin/jung.js:57:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
```
I've tried using sudo in case it was a permissions issues, but no go.
Any ideas?

**Comment by jarofghosts** [comment]
Turns out it was a bug! Fixed in `jung@2.0.3`

**Comment by skinnyjames** [comment]
It's working,  thanks!
